$NetBSD: patch-aa,v 1.1.1.1 2001/02/12 16:15:14 agc Exp $

Get rid of hard-coded PREFIX definition - it gets passed through the environment
in pkgsrc.

--- Makefile	2001/02/12 15:48:04	1.1
+++ Makefile	2001/02/12 15:48:21
@@ -44,7 +44,6 @@
 
 INSTALL=install
 INSTALL_FLAGS=-c -r -m 755 -o bin -g bin
-PREFIX=/usr/local
 FRE=fre
 
 HDRS = options.h ssh_buffer.h ssh_channel.h ssh_client.h ssh_defines.h	\
